packages/mds-web-sockets/wsServer.ts

```diff
@@ -0,0 +1,123 @@
+import logger from '@mds-core/mds-logger'
```

Comment:
Only comment is that we typically avoid mixed case file names. I’d go for `ws-server`